src/kOS/Binding/BindingsUniverse.cs

```diff
@@ -152,6 +152,8 @@ public override void AddTo(SharedObjects shared)
             }
 
             shared.BindingMgr.AddGetter("VERSION", () => Core.VersionInfo);
+
+            shared.BindingMgr.AddGetter("STEERINGMANAGER", () => SteeringManager.GetInstance(shared));
```

Comment:
What is the reason for having this in the universe bindings rather than the flightstats?